examples/webpack2/src/main.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import { render } from 'react-dom'
+import { Button, Container, Header } from 'semantic-ui-react'
```

Comment:
@pruhstal you may have already noticed, but check the latest comments on the PR.  Appears to be a webpack bug.